File: AppGenerateFiles/dbo/FrontModel/Familias.js
//@ts-check
import { EntityClass } from "../../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../../WDevCore/WModules/CommonModel.js";
import { Security_Users }  from './Security_Users.js'
import { Estudiantes_responsables_familias }  from './Estudiantes_responsables_familias.js'
class Familias extends EntityClass {
   /** @param {Partial<Familias>} [props] */
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id;
   /**@type {String}*/ Descripcion;
   /**@type {Boolean}*/ Estado;
   /**@type {Number}*/ Saldo;
   /**@type {Boolean}*/ Actualizado;
   /**@type {Boolean}*/ Aceptacion;
   /**@type {Number}*/ Periodo_aceptacion;
   /**@type {Date}*/ Fecha_actualizacion;
   /**@type {String}*/ Fecha_ultima_notificacion;
   /**@type {Security_Users} ManyToOne*/ Security_Users;
   /**@type {Array<Estudiantes_responsables_familias>} OneToMany*/ Estudiantes_responsables_familias;
}
export { Familias }

File: AppGenerateFiles/dbo/Model/Familias.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Familias : EntityClass {
       [PrimaryKey(Identity = false)]
       public int? Id { get; set; }
       public string? Descripcion { get; set; }
       public bool? Estado { get; set; }
       public Double? Saldo { get; set; }
       public bool? Actualizado { get; set; }
       public bool? Aceptacion { get; set; }
       public int? Periodo_aceptacion { get; set; }
       public DateTime? Fecha_actualizacion { get; set; }
       public string? Fecha_ultima_notificacion { get; set; }
       public int? Id_usuario { get; set; }
       [ManyToOne(TableName = "Security_Users", KeyColumn = "Id_User", ForeignKeyColumn = "Id_usuario")]
       public Security_Users? Security_Users { get; set; }
       [OneToMany(TableName = "Estudiantes_responsables_familias", KeyColumn = "Id", ForeignKeyColumn = "Familia_id")]
       public List<Estudiantes_responsables_familias>? Estudiantes_responsables_familias { get; set; }
   }
}
